test/setup/versions.tf

```diff
@@ -19,11 +19,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.76.0"
+      version = ">= 6.4.0"
     }
     google-beta = {
       source  = "hashicorp/google-beta"
-      version = ">= 4.76.0"
+      version = ">= 6.4.0"
```

Comment:
Can you plz change this to 6.4 instead of 6.4.0?

Reply:
Done.